python/private/pypi/whl_library.bzl

```diff
@@ -561,6 +568,40 @@ if __name__ == "__main__":
     )
     return contents
 
+def _find_namespace_package_files(rctx, install_dir):
```

Comment:
nit, there was a function to generate all of the locations of the namespace package files to be generated, this the inverse, detect them. Should this be in a separate file for easier testability?